File: 217-contains-duplicate/217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    const Sett = new Set()
    
    for (let i = 0; i < nums.length; i++){
        
        if (Sett.has(nums[i])){
            return true
        }
        Sett.add(nums[i])
        
        
    } 
    return false
    
};

File: 217-contains-duplicate/217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        
        
        Sett = set()
        
        for num in nums:
            
            if num in Sett:
                return True
            
            Sett.add(num)